modules/interactive_music/doc_classes/AudioStreamPlaylist.xml

```diff
@@ -35,10 +35,10 @@
 			Fade time used when a stream ends, when going to the next one. Streams are expected to have an extra bit of audio after the end to help with fading.
 		</member>
 		<member name="loop" type="bool" setter="set_loop" getter="has_loop" default="true">
-			If true, the playlist will loop, otherwise the playlist when end when the last stream is played.
+			If [code]true[/code], the playlist will loop, otherwise the playlist will end when the last stream is played.
```

Comment:
```suggestion
			If [code]true[/code], the playlist will loop, otherwise the playlist will end when the last stream is finished.
```
To make it clearer